flink-test-utils-parent/flink-connector-testing/src/main/java/org/apache/flink/connectors/test/common/external/source/TestingSourceOptions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connectors.test.common.external.source;
+
+import org.apache.flink.api.connector.source.Boundedness;
+import org.apache.flink.connector.base.DeliveryGuarantee;
+
+/** Options for configuring the source under testing. */
+public class TestingSourceOptions {
```

Comment:
I'm curious the diff between prefix `testing` and `test`, I suggest we keep same style for readable considerartion.
The module `flink-connector-testing` and `flink-connector-test-utils` can merge to one ? otherwise connector developers need to introduce the two dependencies if they want to write a test with our testing framework.